src/main/scala/com/workday/warp/arbiters/SmartNumberArbiter.scala

```diff
@@ -116,14 +116,29 @@ class SmartNumberArbiter(val lPenalty: Double = WARP_ANOMALY_RPCA_L_PENALTY.valu
         rawResponseTimes
       }
 
-      this.smartNumber(responseTimes, left = 0.0, right = 2.0 * responseTimes.max)
+      this.smartNumber(this.smooth(responseTimes, k = 5), left = 0.0, right = 2.0 * responseTimes.max)
```

Comment:
is `k=5` the suggested starting number of entries for smoothing?

Reply:
ill add some doc, cool. theres not a default value for `k`, i used the arg literal syntax at the call site.

maybe add a configuration parameter for this. i'd like a better name than `k` also